### PR TITLE
Add Online field to pool list output

### PR DESCRIPTION
**Pull request by prashanthpai**
Updates #742 

Signed-off-by: Prashanth Pai <ppai@redhat.com>

### CODE REVIEW

File: glustercli/cmd/peer.go [hunk 1]
```diff
@@ -108,9 +109,9 @@ func peerStatusHandler(cmd *cobra.Command) {
 		failure("Failed to get Peers list", err, 1)
 	}
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"ID", "Name", "Peer Addresses"})
+	table.SetHeader([]string{"ID", "Name", "Peer Addresses", "Online"})
```
[the remark this answers is not in the document]
Comment (prashanthpai) [reply glustercli/cmd/peer.go]: I want the field to be "true" or "false". Status is true doesn't make much sense.

File: glustercli/cmd/peer.go [hunk 2]
```diff
@@ -108,9 +109,9 @@ func peerStatusHandler(cmd *cobra.Command) {
...
 	for _, peer := range peers {
-		table.Append([]string{peer.ID.String(), peer.Name, strings.Join(peer.PeerAddresses, ",")})
+		table.Append([]string{peer.ID.String(), peer.Name, strings.Join(peer.PeerAddresses, ","), strconv.FormatBool(peer.Online)})
```
[the remark this answers is not in the document]
Comment (prashanthpai) [reply glustercli/cmd/peer.go]: We do not maintain persistent connections to other glusterd2 instances. It'll be inaccurate to say that it's connected or disconnected.

[the remark this answers is not in the document]
Comment (aravindavk) [reply glustercli/cmd/peer.go]: ok

File: glustercli/cmd/peer.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (aravindavk) [reply glustercli/cmd/peer.go]: Yes/No looks good compared to true/false

[the remark this answers is not in the document]
Comment (prashanthpai) [reply glustercli/cmd/peer.go]: Done.

**Comment by prashanthpai** [comment]
```sh
[ppai@gd2-1 ~]$ glustercli pool list
+--------------------------------------+-------+-----------------+--------+
|                  ID                  | NAME  | PEER ADDRESSES  | ONLINE |
+--------------------------------------+-------+-----------------+--------+
| 84b419be-7300-4cb2-be42-c28908e61d6b | gd2-1 | 127.0.0.1:24008 | yes    |
+--------------------------------------+-------+-----------------+--------+
```

**Comment by prashanthpai** [comment]
retest this please

**Comment by prashanthpai** [comment]
retest this please

**Comment by prashanthpai** [comment]
retest this please

**Comment by prashanthpai** [comment]
retest this please

**Pull request merged**